src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.cs

```diff
@@ -1,37 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Reflection;
-
 namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
-        private const string FrameworkName = ".NET";
-        private static string? s_frameworkDescription;
         private static string? s_runtimeIdentifier;
 
-        public static string FrameworkDescription
-        {
-            get
-            {
-                if (s_frameworkDescription == null)
-                {
-                    ReadOnlySpan<char> versionString = typeof(object).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
-
-                    // Strip the git hash if there is one
-                    int plusIndex = versionString.IndexOf('+');
-                    if (plusIndex >= 0)
-                    {
-                        versionString = versionString.Slice(0, plusIndex);
-                    }
-
-                    s_frameworkDescription = !versionString.Trim().IsEmpty ? $"{FrameworkName} {versionString}" : FrameworkName;
-                }
-
-                return s_frameworkDescription;
-            }
-        }
+        /// <summary>
+        /// Gets the name of the .NET installation on which an app is running.
+        /// </summary>
+        public static string FrameworkDescription { get; } = ".NET " + Environment.ProductVersionWithLabel;
```

Comment:
```suggestion
        public static string FrameworkDescription => ".NET " + Environment.ProductVersionWithLabel;
```
This should turn into a constant string literal, No need to cache it.